src/pspm_pupil_correct_eyelink.m

```diff
@@ -178,25 +155,14 @@
   gaze_y_chan = 'gaze_y_r';
 end
 
-[lsts, ~, gaze_x_data] = pspm_load_data(fn, gaze_x_chan);
+[lsts, gaze_x_data] = pspm_load_channel(fn, gaze_x_chan, 'gaze');
```

Comment:
should this be `gaze_x` instead of `gaze`? I saw in `pspm_load_channel`, the allowed gaze-related channels are 'gaze_x' and 'gaze_y'.